modules/monitoring-getting-detailed-information-about-a-target.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-targets.adoc
+
+[id="getting-detailed-information-about-a-target_{context}"]
+= Getting detailed information about a target
+
+The Targeting UI provides detailed information about HTTP endpoints that are targeted for metrics scraping.
+
+.Prerequisites
+
+* You have access to the cluster as an administrator for the project for which you want to view metrics targets.
```

Comment:
This prerequisite also applies to the targets list page. I think including this note here makes it look like it only applies to the details page.

Reply:
Good point. I added it to the other section.